packages/e2e-tests/test/fixtures/curl.js

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env node
 'use strict';
-const fetch = require('node-fetch');
+import fetch from 'node-fetch';
```

Comment:
```
$ ./packages/e2e-tests/test/fixtures/curl.js 
/Users/addaleax/src/mongosh/packages/e2e-tests/test/fixtures/curl.js:3
const fetch = require('node-fetch');
              ^

Error [ERR_REQUIRE_ESM]: require() of ES Module /Users/addaleax/src/mongosh/packages/e2e-tests/node_modules/node-fetch/src/index.js from /Users/addaleax/src/mongosh/packages/e2e-tests/test/fixtures/curl.js not supported.
Instead change the require of index.js in /Users/addaleax/src/mongosh/packages/e2e-tests/test/fixtures/curl.js to a dynamic import() which is available in all CommonJS modules.
    at Object.<anonymous> (/Users/addaleax/src/mongosh/packages/e2e-tests/test/fixtures/curl.js:3:15) {
  code: 'ERR_REQUIRE_ESM'
}

Node.js v20.11.1
```

probably needs to be dynamically imported inside the main function here :)

Reply:
I think I already fixed that

Reply:
Fwiw, this was with the most recent version of this PR, so it seems like it's still failing here? I don't know how it would work – this is a `.js` file and those don't support top-level `import`. You could try renaming this to `.mjs`, if you want, that might be a "cool" thing to do to fix it :)

Reply:
hmm I didn't see this error in the recent version, but there are similar errors from mocha, which I'm currently trying to fix.